markup.js

```diff
@@ -271,4 +203,30 @@ function buildKeyboard (buttons, options) {
   return result
 }
 
+function tag (name, params) {
+  return {
+    open: params
+      ? `<${name} ${Object.entries(params).map(([key, value]) => `${key}="${value}"`).join(' ')}>`
```

Comment:
Attribute values should be escaped as well:
```suggestion
      ? `<${name} ${Object.entries(params).map(([key, value]) => `${key}="${value.replace(/[<&"]/g, escapeChar)}"`).join(' ')}>`
```

On 2nd thought, no need to escape apostrophe.